Lib/urllib/request.py

```diff
@@ -2008,15 +2008,18 @@ def _proxy_bypass_winreg_override(host, override):
     from fnmatch import fnmatch
 
     host, _ = _splitport(host)
-    proxy_override = override.split(';')
-    for test in proxy_override:
-        test = test.strip()
```

Comment:
This code is clearer. If the fix is to skip empty entries, just add ``if not test: continue`.